### ISSUE TITLE
Survival prediction models: an introduction to discrete-time modeling

### ISSUE THREAD

**Issue by cmclean5** [cmclean5/PublicHealthModels#15]
## TL;DR

Paper is presented as a tutorial for building survival prediction models using discrete-time modeling. Used for it's weighted Brier Score (BS) algorithm.

### Paper Link

https://bmcmedresmethodol.biomedcentral.com/articles/10.1186/s12874-022-01679-6#Sec11

### Author/Institution

Krithika Suresh (University of Colorado) Cameron Severn (University of Colorado) & Debashis Ghosh (University of Colorado)

## Overview

Paper discusses building and testing discrete-time prediction models. We additionally make available R code for implementing this process and for comparing the predictive performance to common continuous-time survival models. The primary goal is to provide applied statisticians with resources for training and evaluating discrete-time prediction models to encourage their use in medical research for personalized decision-making.

## Cite

Suresh, K., Severn, C. & Ghosh, D. Survival prediction models: an introduction to discrete-time modeling. BMC Med Res Methodol 22, 207 (2022). https://doi.org/10.1186/s12874-022-01679-6
